src/theory/inference_id.h

```diff
@@ -172,6 +172,9 @@ enum class InferenceId
   ARITH_NL_ICP_CONFLICT,
   // propagation / contraction of variable bounds from icp
   ARITH_NL_ICP_PROPAGATION,
+  //-------------------- ff inference
+  // arith ff
+  ARITH_FF,
```

Comment:
Following the naming convention, this should be something like `FF_LEMMA` (prefixed by the theory `FF_`).

Reply:
:+1: